boxoffice/models/discount_policy.py

```diff
@@ -165,6 +165,20 @@ def line_items_count(self):
         from ..models import LineItem, LINE_ITEM_STATUS
         return self.line_items.filter(LineItem.status == LINE_ITEM_STATUS.CONFIRMED).count()
 
+    @classmethod
+    def is_valid_access_coupon(cls, item, code_list):
```

Comment:
This method needs documentation on why there's a search through both `DiscountPolicy` and `DiscountCoupon` models.

Reply:
Added documentation.